integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -117,6 +117,20 @@
                         }
                     },
                     {
+                        bidder: 'sonobi',       //  New format
+                        params: {
+                            dom_id: 'PER SLOT', //  <String> dom Id

```

Comment:
@Studnicky Is there a chance you could provide an example of what these parameters would look like?  For example, is `dom_id` something like `ABC124` or `leaderboard `?